translations/ui-data-import/en.json

```diff
@@ -39,6 +39,8 @@
   "duplicate": "Duplicate",
   "endOfRecord": "End of record",
   "description": "Description",
+  "details": "Details",
```

Comment:
The "Details" was added to en.json (Please, take a look at "settings.matchProfiles.details") Please, remove dublication ("settings.matchProfiles.details" or "details") and update the needed parts of code.

Reply:
Done